include/cutlass/epilogue/collective/builders/xe_builder.inl

```diff
@@ -118,10 +118,20 @@ namespace detail {
         CopyOpG2R
       >;
   };
-}
 
-  // Intel epilogue builder
+template <typename T>
+struct is_rowmajorish {
```

Comment:
I think this should be called `is_MN_majorish` or something? Especially since matrix C has dimensions `(M, N)`, and I think `is_stride_leftmost` is actually `ColumnMajor`, isn't it? Disclaimer:  :sleeping: :coffee: